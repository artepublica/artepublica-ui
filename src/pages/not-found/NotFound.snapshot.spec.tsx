import { create } from 'react-test-renderer';

import NotFound from './NotFound';

jest.mock('@react-navigation/native', () => {
    return {
        ...jest.requireActual('@react-navigation/native'),
        useNavigation: () => ({
            dispatch: jest.fn(),
        }),
    };
});

describe('<NotFound />', () => {
    it('default snapshot', () => {
        const json = create(<NotFound />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
