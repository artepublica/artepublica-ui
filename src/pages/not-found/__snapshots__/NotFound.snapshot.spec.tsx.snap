// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> default snapshot 1`] = `
<RNCSafeAreaView
  edges={
    Array [
      "bottom",
      "left",
      "right",
      "top",
    ]
  }
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "fontFamily",
        },
        undefined,
      ]
    }
  >
    404 not found
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "borderColor": "white",
          "borderRadius": 8,
          "borderWidth": 1,
          "color": "white",
          "fontFamily": "fontFamily",
          "padding": 16,
        },
        false,
      ]
    }
  >
    <Text>
      go to home
    </Text>
  </View>
</RNCSafeAreaView>
`;
